monkey-runner/build.gradle

```diff
@@ -0,0 +1,24 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath gradlePlugins.gradle
+        classpath gradlePlugins.gradleAndroidCommandPlugin
+        classpath gradlePlugins.bintrayRelease
+    }
+}
+
+apply from: 'dependencies.gradle'
```

Comment:
```groovy
apply from: 'dependencies.gradle'
buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath gradlePlugins.gradle
        classpath gradlePlugins.gradleAndroidCommandPlugin
        classpath gradlePlugins.bintrayRelease
    }
}
allprojects {
    repositories {
        jcenter()
    }
    version = '0.0.1'
}
```

This didn't work - "Could not get unknown property 'gradlePlugins' for object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler".